### PR TITLE
Pull Request adicionando as novas features

### PR DESCRIPTION
**Pull request by sslp23**

### CODE REVIEW

File: features/ExportarPlanilhaEstoque.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature: Exportar Planilha de Estoque
+    Como gestor, desejo exportar a planilha de estoque para o meu computador
+
```
Comment (sbjsouza) [inline features/ExportarPlanilhaEstoque.feature:3]: I Think that we can remove this extra white space

```suggestion
```

File: features/GerenciamentoDeVendas.feature [hunk 2]
```diff
@@ -0,0 +1,73 @@
+Feature: Gerenciamento de vendas
+    Como vendedor, gostaria de registrar minhas vendas e o valor no sistema, verificar se há estoque do produto, ver quais são as características do produto e opções de pagamento
+
+Scenario: Registrar uma venda
+    Given “Samuel” está definido na base de dados como “Vendedor”
+    And “Samuel” está na “Página de Gerenciamento de Vendas”
+    When eu seleciono “produto vendido” 
+    Then eu sou direcionado para a página “Registro de vendas”
+    And  eu vejo que o título da página é “Informe suas vendas”
```
Comment (sbjsouza) [inline features/GerenciamentoDeVendas.feature]: I think that we use `And` statement to actions

File: features/GerenciamentoDeVendas.feature [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    Then o sistema mostra a quantidade de “produto 1” no estoque
+
+Scenario: Verificar informações do produto
+    Given “Samuel” está definido na base de dados como “Vendedor”
+    And “Samuel” está na “Página de Gerenciamento de Vendas”
+    When eu seleciono “informação sobre produtos” 
+    And eu seleciono produto “produto 1”
+    Then eu sou direcionado para a página “informações”
+    And eu seleciono “visualizar informações do produto 1” 
+    Then o sistema mostra a descrição “TV 45 Polegadas LG” e avaliação “4,5/5” do produto “produto 1”
```
Comment (sbjsouza) [inline features/GerenciamentoDeVendas.feature]:
```suggestion
    Then o sistema retorna os campos "descrição" com o valor “TV 45 Polegadas LG” e "avaliação" com o valor “4,5/5”, associado ao “produto 1” 
```

**Review by sbjsouza — CHANGES_REQUESTED** [review]
Amazing work, keep going  🎉! But don't forget to check some comments in my code review

**Comment by sslp23** [comment]
Os 3 problemas foram resolvidos

**Pull request closed without merging**